### ISSUE TITLE
Resumo dos dias permanecidos no país no período de 12 meses

### ISSUE THREAD

**Issue by lnmunhoz** [sampavalley/liontax-calendar#37]
Mostrar card com o resumo dos dias permanecidos no país no período de 12 meses a contar da data de entrada **(regra de negócio)** .
> Somente se for Brasil

**Comment by lnmunhoz** [comment]
Na situação que o usuário adicionou uma data de chegada no Brasil a mais 12 meses atrás, porém se o período de permanência estiver dentro da faixa de 12 meses. (Ex. Chegada em 2 de fevereiro de 2015 e permaneceu ate 10 de fevereiro de 2015. Data de hoje 5 de fevereiro 2016.) Devemos fazer a contagem dos dias que ainda permanecem na régua?

**Comment by Lionpam** [comment]
Não. Nesse caso elimina-se o período de 2 de fevereiro a 10 de fevereiro e inicia a contagem da data de entrada seguinte. Por exemplo 1 de março. Eu subi no Google drive um Excel com regra de contagem e uma mensagem específica para uma situação de conflito não esclarecida pela lei. Por favor, dê uma olhada na planilha e se tiver alguma
Duvida me avise. Obrigada!
